README.md

```diff
@@ -546,6 +547,26 @@ Additionally an optional string parameter `system_prompt` can be specified in `/
 
 Activate token quota limits for the service by adding a new configuration structure into the configuration file. That structure should be added into `ols_config` section. It has the following format:
 
+
+
+### Tokens and token quota limits
+
+Tokens are small chunks of text, which can be as small as one character or as large as one word. Tokens are the units of measurement used to quantify the amount of text that the service sends to, or receives from, a large language model (LLM). Every interaction with the Service and the LLM is counted in tokens.
+
+Token quota limits define the number of tokens that can be used in a certain timeframe. Implementing token quota limits helps control costs, encourage more efficient use of queries, and regulate system demands. In a multi-user configuration, token quota limits help provide equal access to all users ensuring everyone has an opportunity to submit queries.
+
+### Quota limiter features
+
+It is possible to limit quota usage per user or per service or services (that typically run in one cluster). Each limit is configured as separate so called _quota limiter_. It can be of type `user_limiter` or `cluster_limiter` (which is name that makes sense in OpenShift deployment). There are three configuration options for each limiter:
+
+1. `period` specified in human readable form, see https://www.postgresql.org/docs/current/datatype-datetime.html#DATATYPE-INTERVAL-INPUT for all possible options. When the period is reached, quota is reset or increased
```

Comment:
"human readable" -> "a human-readable".
"When the period is reached" -> "When the end of the period is reached"

Reply:
fixed